src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -306,10 +305,7 @@ create_reference_table(PG_FUNCTION_ARGS)
 	 */
 	EnsureRelationKindSupported(relationId);
 
-	workerNodeList = ActivePrimaryNodeList(ShareLock);
-	workerCount = list_length(workerNodeList);
-
-	/* if there are no workers, error out */
+	workerCount = ActivePrimaryWorkerNodeCount();
 	if (workerCount == 0)
```

Comment:
This is create_reference_table, shouldn't we include the coordinator?

Reply:
Do we still need to error out here?

Reply:
done